obsei/preprocessor/text_cleaner.py

```diff
@@ -0,0 +1,108 @@
+import logging
+from typing import Any, Dict, List, Optional
+import string
+import re
+from unicodedata import normalize as unicode_decode
+
+import nltk
+from nltk.corpus import stopwords
+from nltk.stem import PorterStemmer
+from nltk.tokenize import word_tokenize
+
+from pydantic import PrivateAttr
+
+from obsei.analyzer.base_analyzer import AnalyzerRequest
+from obsei.preprocessor.base_text_cleaner import (
+    BaseTextCleaner,
+    BaseTextCleanerConfig,
+)
+
+logger = logging.getLogger(__name__)
+nltk.download('punkt')
```

Comment:
See if we could do something like this https://github.com/lalitpagaria/obsei/blob/master/obsei/analyzer/pii_analyzer.py#L83

In dockerfile generally we download model or related files and then create image.
So idea is to download only if not exist.

Reply:
nltk already does that check, so it won't redownload if file is already there.